README.md

```diff
@@ -122,7 +122,8 @@ Example call:
 ```js
 import { client } from '@passwordless-id/webauthn' 
 
-const registration = await client.register("Arnaud", "a7c61ef9-dc23-4806-b486-2428938a547e", {
```

Comment:
I was confused when first reading what a7.. was referring to, as I did not connect it with the `const challenge = ` just before :)